uoftscrapers/scrapers/calendar/utm.py

```diff
@@ -4,14 +4,75 @@
 import json
 import os
 import requests
+import datetime
+now = datetime.datetime.now()
 
 
 class UTMCalendar:
+    '''Scraper for Important dates from UTM calendar found at https://www.utm.utoronto.ca/registrar/important-dates
+        '''
 
-    host = 'http://www.artsandscience.utoronto.ca/ofr/calendar/'
-
+    summerLink = 'http://m.utm.utoronto.ca/importantDates.php?mode=full&session={0}5&header='
+    fallLink = 'http://m.utm.utoronto.ca/importantDates.php?mode=full&session={0}9&header='
+    sessionLinks = [summerLink, fallLink]
+    currentSession = "Summer"
```

Comment:
are there any advantages to scraping it instead of just having it as is? thanks.